world/blocks.md

```diff
@@ -26,9 +26,15 @@ Block tnt = Block.TNT.withProperty("unstable", "true");
 instance.setBlock(0, 41, 0, tnt);
 ```
 
-## Registry
+## BlockEntry
 
-Each block has unique data which can be retrieved with `Block#registry()`.
+Each block is associated with a `BlockEntry`,which store some **unmodifiable **(not immutable) data generated by **vanilla**.
+
+For example, `BlockEntry#explosionResistance()` return the explosion resistance in vanilla,and it's unmodifiable.However,it is just a value and will not really affect the explosion.
+
+`BlockEntry` is unmodifiable by now, though we have the plan to make it modifiable in the future.
+
+`BlockEntry` could be retrieved with `Block#registry()`.
```

Comment:
_can_ be retrieved

Reply:
Also: is unmodifiable *for* now
Might also help: thought *we plan* to make it modifiable in the future.